doc/developers/contributing.rst

```diff
@@ -383,6 +383,17 @@ documentation with the maths makes it more friendly towards
 users that are just interested in what the feature will do, as
 opposed to how it works "under the hood".
 
+You may also be asked to show your changes when it's built. When you create
+a pull request or make changes in an existing one modifying the docs, circleci
+automatically builds them. Thus, you can easily view your changes in the built
+artifacts using the following formula:
+
+``https://{BUILD_NUMBER}-843222-gh.circle-artifacts.com/0//home/ubuntu/scikit-learn/doc/_build/html/stable/index.html``
```

Comment:
With https://github.com/scikit-learn/scikit-learn.github.io/pull/5, we could shorten this URL to http://scikit-learn.org/circle?{BUILD_NUMBER}